### PR TITLE
BaSP-TG-38: Tasks add shared components

### PR DESCRIPTION
**Pull request by PilarFernandezC**
- Add shared components (Button, Modal, Table, Row)
- Modify form so edit functionality work correctly
- Add routes to Layout (Edit and Delete functionalities)

Success and error messages in edit and create are shown in alerts (we will change it to modals)

### CODE REVIEW

File: src/Components/Shared/Row/index.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          );
+        } else {
+          return <td key={index}>{data[header]}</td>;
+        }
+      })}
+    </tr>
+  );
+};
+
+export default Row;
```
Comment (Juanoauil) [inline src/Components/Shared/Row/index.jsx:38]: this component was not already merge?

**Review by Juanoauil — COMMENTED** [review]
please resolve conflict when you create a pr

File: src/Components/Tasks/form.jsx [hunk 2]
```diff
@@ -28,11 +39,18 @@ const Form = () => {
   };
 
   const redirect = () => {
-    window.location.assign('/tasks');
+    history.push('/tasks');
```
Comment (DamianPalavecino) [inline src/Components/Tasks/form.jsx:42]:
```suggestion
    history.goBack();
```

**Review by Juanoauil — APPROVED** [review]

File: src/Components/Tasks/form.jsx (hunk 2 above)
Comment (PilarFernandezC) [reply src/Components/Tasks/form.jsx]: Done

**Pull request merged**